src/e2e-app/mdc-slider/mdc-slider-e2e.ts

```diff
@@ -10,7 +10,20 @@ import {Component} from '@angular/core';
 
 @Component({
   selector: 'mdc-slider-e2e',
-  template: `<mat-slider></mat-slider>`,
+  template: `
+    <mat-slider id="standard-slider">
+      <input aria-label="Standard slider" matSliderThumb>
+    </mat-slider>
+
+    <mat-slider id="disabled-slider" disabled>
+      <input aria-label="Disabled slider" matSliderThumb>
+    </mat-slider>
+
+    <mat-slider id="range-slider">
+      <input aria-label="Standard Range slider" matSliderStartThumb>
+      <input aria-label="Standard Range slider" matSliderEndThumb>
```

Comment:
Shouldn't the different thumbs have unique labels?

Reply:
That's what I thought, too, but when I checked what the MDC team was doing this is how they had their examples set up.

https://github.com/material-components/material-components-web/tree/master/packages/mdc-slider#continuous-range-slider

Maybe our friendly neighborhood a11y expert @jelbourn will know what the correct setup is?

Reply:
AFAIK they should have different labels, I wouldn't consider their demo to be normative